src/components/HotPricesSwiper/HotPricesSwiper.tsx

```diff
@@ -0,0 +1,31 @@
+import { FC } from 'react';
+import { Product } from '@/types/Product';
+import { ProductSlider } from '../ProductSlider/ProductSlider';
+import { DeviceCategory } from '@/enums';
+
+interface ProductSliderProps {
+  products: Product[];
+}
+
+export const HotPricesSwiper: FC<ProductSliderProps> = ({ products }) => {
+  function getTopDiscountProductsByCategory(productsArr: Product[]): Product[] {
```

Comment:
Why not a helper and inside component?
It will create this function each render, it is not good